### PR TITLE
Fix post cra2

### PR DESCRIPTION
**Pull request by harunurhan**

### CODE REVIEW

File: src/common/components/ListItemAction/ListItemAction.jsx [hunk 1]
```diff
@@ -6,6 +6,8 @@ import { Button } from 'antd';
 import IconText from '../IconText';
 import './ListItemAction.scss';
 
+const CLASS_NAME = '__ListItemAction__';
```
Comment (drjova) [inline src/common/components/ListItemAction/ListItemAction.jsx]: maybe something related to a button?

Comment (harunurhan) [reply src/common/components/ListItemAction/ListItemAction.jsx]: this is how we name root node class for a component though.

Comment (drjova) [reply src/common/components/ListItemAction/ListItemAction.jsx]: in this case is just for the button, no?

Comment (harunurhan) [reply src/common/components/ListItemAction/ListItemAction.jsx]: it's the root node for this component though.

**Pull request merged**